executor/aggregate.go

```diff
@@ -51,15 +51,32 @@ type baseHashAggWorker struct {
 	aggFuncs     []aggfuncs.AggFunc
 	maxChunkSize int
 	stats        *AggWorkerStat
+
+	memTracker *memory.Tracker
+	BInMap     *int // indicate there are 2^BInMap buckets in Golang Map.
```

Comment:
Why use a pointer here?

Reply:
My fault.  I think we should use *baseHashAggWorker to call getPartialResult..